lib/task-data/tasks/install-coreos.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     implementsTask: 'Task.Base.Os.Install',
     options: {
         // NOTE: user/pass aren't used by the coreos installer at the moment,
-        // but they are required values
+        // but they are required values for Task.Base.Os.Install
```

Comment:
so... apparently it does!, huh - will update the PR to just remove that stuff since it doesn't use it OR require it